### ISSUE TITLE
Talkback should be able to detect progress bar changes through sound feedback

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#256]
```
Talkback should through sound feedback report the movement of a progress bar in 
lets say a download from the play store. I would assume that the sounds would 
be an assenting tone as the progress bar increases in completion.
```

Original issue reported on code.google.com by `jmwalto...@gmail.com` on 1 May 2013 at 6:03

**Comment by GoogleCodeExporter** [comment]
```
We currently do this.  To avoid unwanted feedback, you do have to navigate to 
the seek control before it will begin speaking status updates.
```

Original comment by `caseybur...@google.com` on 29 May 2013 at 7:26
- Changed state: **Invalid**
